README.md

```diff
@@ -18,13 +18,18 @@ In order to run the code here, you need to install the following software
 * GCC (C- Compiler)
 * barectf
```

Comment:
In this case I think it is also important to mention how to get barectf on an ubuntu machine 14.04 and 16.04
sudo apt-add-repository ppa:lttng/barectf
sudo apt-get update
sudo apt-get install python3-barectf
Then see https://github.com/efficios/barectf for others.